src/components/CommentCard.jsx

```diff
@@ -14,6 +59,12 @@ export default function CommentCard({ comment }) {
       <button>+</button>
       {comment.votes}
       <button>-</button>
+
+      {deleteBtn && (
+        <button value={deleteBtn} onClick={handleDelete}>
+          X
```

Comment:
I'd change this to say delete instead of just X so the user knows exactly what it's going to do